src/lib/Web3Function.ts

```diff
@@ -1,17 +1,14 @@
 import { BigNumber } from "@ethersproject/bignumber";
+import { diff } from "deep-object-diff";
 import { Web3FunctionHttpServer } from "./net/Web3FunctionHttpServer";
+import "./polyfill/XMLHttpRequest";
+import { Web3FunctionMultiChainProvider } from "./provider/Web3FunctionMultiChainProvider";
 import {
   Web3FunctionContext,
   Web3FunctionContextData,
 } from "./types/Web3FunctionContext";
+import { Web3FunctionEvent } from "./types/Web3FunctionEvent";
 import { Web3FunctionResult } from "./types/Web3FunctionResult";
-import {
-  Web3FunctionEvent,
-  Web3FunctionStorage,
-} from "./types/Web3FunctionEvent";
-import objectHash = require("object-hash");
```

Comment:
if `object-hash` is not used anymore, we should remove it from the package dependencies

Reply:
It's removed already

Reply:
removed in c42814f3dbdd8df8e2b5bcbf73eaaed5472dd44c